plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -171,6 +172,20 @@ public void testTopNPushdown()
                     .ordered()
                     .isFullyPushedDown();
         }
+
+        // TopN over LEFT join (enforces SINGLE TopN cannot be pushed below OUTER side of join)
+        // We expect PARTIAL TopN on the LEFT side of join to be pushed down.
+        assertThat(query("" +
+                "SELECT * " +
+                "FROM nation n LEFT JOIN region r ON n.nationkey = r.regionkey " +
+                "ORDER BY n.name DESC " +
+                "LIMIT 3"))
```

Comment:
- merge these lines
- is "DESC" important? perhaps remove
- is ordering on a varchar column name important? this would go into `testCaseSensitiveTopNPushdown`, so let's use a numeric column, eg nationkey for order by (and use n.regionkey for join)